source/_integrations/keenetic_ndms2.markdown

```diff
@@ -46,6 +58,20 @@ interface:
   required: false
   default: Home
   type: string
+try_hotspot:
+  description: Use `ip hotspot` data for device detection. This is the most accurate and preferred way of devices detection. If enabled and available other options below ignored.
+  required: false
+  default: true
+  type: boolean
+include_arp:
+  description: Use ARP data for device detection. This will show devices having constant traffic through the router. Ignored if hotspot data is available.
+  required: false
+  default: true
+  type: boolean
+include_associated:
+  description: Use WiFi AP associations data for device detection. This will show devices actually connected to AP on the device.
+  required: false
+  default: true
+  type: boolean
```

Comment:
As per ADR-0010, we don't allow changes to the YAML configuration anymore:

<https://github.com/home-assistant/architecture/blob/master/adr/0010-integration-configuration.md#decision>